File: Frontend/src/types/bowler.ts
import type { team } from "./team";

export type bowler = {
    bowlerID: number,
    bowlerLastName: string,
    bowlerFirstName: string,
    bowlerMiddleInit: string,
    bowlerAddress: string,
    bowlerCity: string,
    bowlerState: string,
    bowlerZip: string,
    bowlerPhoneNumber: string,
    teamID: number,
    team: team
};

File: Mission10_Assignment/Data/Bowler.cs
using Microsoft.Extensions.Diagnostics.HealthChecks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Mission10_Assignment.Data
{
    public class Bowler
    {
        [Key]
        public int BowlerID { get; set; }
        public string BowlerLastName { get; set; }
        public string BowlerFirstName { get; set; }
        public char BowlerMiddleInit { get; set; }
        public string BowlerAddress { get; set; }
        public string BowlerCity { get; set; }
        public string BowlerState { get; set; }
        public string BowlerZip { get; set; }
        public string BowlerPhoneNumber { get; set; }
        public int TeamID { get; set; }
        [ForeignKey("TeamID")]
        public Team Team { get; set; }
    }
}
